src/bilder/images/edxapp/templates/ansible_vars.j2.yml

```diff
@@ -0,0 +1,154 @@
+EDXAPP_FILE_UPLOAD_STORAGE_PREFIX: "{{ edxapp_upload_storage_prefix }}"
+EDXAPP_IMPORT_EXPORT_BUCKET: {{ bucket_prefix }}-storage-{{ purpose }}-{{ environment }}
+EDXAPP_LMS_BASE: "{{ LMS_DOMAIN }}"
+EDXAPP_LMS_ISSUER: https://{{ context.lms_domain }}/oauth2
+EDXAPP_LMS_ROOT_URL: "https://{{ LMS_DOMAIN }}"
+EDXAPP_LMS_SITE_NAME: "{{ purpose_data.domains.lms }}"
+EDXAPP_LOGIN_REDIRECT_WHITELIST: {{ edxapp_domains|tojson }}
+EDXAPP_GRADE_BUCKET: {{ bucket_prefix }}-grades-{{ purpose }}-{{ environment }}
+EDXAPP_GRADE_ROOT_PATH: {{ edxapp_aws_grades_root_path }}
+EDXAPP_MKTG_URLS:
+  ROOT: "{{ env_data.purposes[purpose].domains.lms }}"
+  ABOUT: "{{ heroku_env }}/about-us"
+  CONTACT: "https://xpro.zendesk.com/hc/en-us/requests/new"
+  HONOR: "honor-code"
+  PRIVACY: "privacy-policy"
+  TOS: "terms-of-service"
+EDXAPP_VIDEO_IMAGE_SETTINGS:
+  VIDEO_IMAGE_MAX_BYTES : 2097152
+  VIDEO_IMAGE_MIN_BYTES : 2048
+  STORAGE_CLASS: 'storages.backends.s3boto.S3BotoStorage'
+  STORAGE_KWARGS:
+    bucket: {{ bucket_prefix }}-edx-video-upload-{{ purpose }}-{{ environment }}
+  DIRECTORY_PREFIX: 'video-images/'
+EDXAPP_VIDEO_TRANSCRIPTS_SETTINGS:
+  VIDEO_TRANSCRIPTS_MAX_BYTES : 3145728
+  STORAGE_CLASS: 'storages.backends.s3boto.S3BotoStorage'
+  STORAGE_KWARGS:
+    bucket: {{ bucket_prefix }}-storage-{{ purpose }}-{{ environment }}
+  DIRECTORY_PREFIX: 'video-transcripts/'
+
+
+# Static Values
+COMMON_ENABLE_AWS_ROLE: false
+COMMON_ENABLE_BASIC_AUTH: false
+COMMON_ENABLE_MINOS: false
+COMMON_ENABLE_NEWRELIC: false
+COMMON_ENABLE_NEWRELIC_INFRASTRUCTURE: false
+EDXAPP_ANALYTICS_DASHBOARD_URL: !!null
+EDXAPP_BUGS_EMAIL: {{ smtp_config.data.support_email }}
+EDXAPP_CACHE_BACKEND: "django_redis.cache.RedisCache"
+EDXAPP_CELERY_BROKER_HOSTNAME: edx-redis.service.consul
+EDXAPP_CELERY_BROKER_TRANSPORT: 'redis'
+EDXAPP_CELERY_USE_SSL: true
+EDXAPP_CMS_MAX_REQ: 1000
+EDXAPP_CMS_NGINX_PORT: 80
+EDXAPP_CMS_SSL_NGINX_PORT: 443
+EDXAPP_COMMENTS_SERVICE_URL: "http://edx-forum.service.consul:4567"
+EDXAPP_CONTACT_EMAIL: {{ smtp_config.data.support_email }}
+EDXAPP_CUSTOM_COURSES_EDX: true
+EDXAPP_DEFAULT_FEEDBACK_EMAIL: "{{ smtp_config.data.support_email }}"
+EDXAPP_DEFAULT_FILE_STORAGE: 'storages.backends.s3boto.S3BotoStorage'
+EDXAPP_DEFAULT_FROM_EMAIL: "{{ smtp_config.data.support_email }}"
+EDXAPP_EMAIL_HOST: {{ smtp_config.data.relay_host }}
+EDXAPP_EMAIL_PORT: {{ smtp_config.data.relay_port }}
+EDXAPP_EMAIL_USE_TLS: true
+EDXAPP_ENABLE_COMBINED_LOGIN_REGISTRATION: true
+EDXAPP_ENABLE_CORS_HEADERS: true
+EDXAPP_ENABLE_CROSS_DOMAIN_CSRF_COOKIE: true
+EDXAPP_ENABLE_CSMH_EXTENDED: true
+EDXAPP_ENABLE_DISCUSSION_HOME_PANEL: true
+EDXAPP_ENABLE_GRADE_DOWNLOADS: true
+EDXAPP_ENABLE_LTI_PROVIDER: true
+EDXAPP_ENABLE_MOBILE_REST_API: true
+EDXAPP_ENABLE_OAUTH2_PROVIDER: true
+EDXAPP_ENABLE_READING_FROM_MULTIPLE_HISTORY_TABLES: false
+EDXAPP_ENABLE_SPECIAL_EXAMS: true
+EDXAPP_ENABLE_THIRD_PARTY_AUTH: true
```

Comment:
What's the plan here? What auths are we planning to support, and what add'l configuration do we need?